packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -407,11 +407,13 @@ export class ModifierOpcode extends Opcode {
   evaluate(vm: VM) {
     let { manager } = this;
     let stack = vm.stack();
-    let { constructing: element, updateOperations } = stack;
+    let { constructing: element } = stack;
     let args = this.args.evaluate(vm);
     let dynamicScope = vm.dynamicScope();
+    let modifier = manager.create(args, dynamicScope);
+
+    vm.env.didCreateModifier(modifier, manager, element as FIX_REIFICATION<Element>);
 
-    let modifier = manager.install(element as FIX_REIFICATION<Element>, args, updateOperations, dynamicScope);
     let destructor = manager.getDestructor(modifier);
 
     if (destructor) {
```

Comment:
Can't comment below this line, but we don't need pass the args and dynamicScope to the updating opcode anymore